### PR TITLE
add JBDS 9.1 Beta2

### PR DESCRIPTION
**Pull request by nickboldt**
add JBDS 9.1 Beta2

### CODE REVIEW

File: _config/products.yml [hunk 1]
```diff
@@ -78,6 +78,33 @@ devstudio:
         update_site_url: https://devstudio.redhat.com/9.0/development/updates/
         #marketplace_url: https://marketplace.eclipse.org/node/1616973
         #marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2015-02-02
+        supported_devstudio_is_version: 9.0.0.Alpha2
```
Comment (alexeykazakov) [inline _config/products.yml]: A little bit late for 9.0.0.Alpha2, no? ;-)
And please fix the release date too.

Comment (alexeykazakov) [inline _config/products.yml]: Ok. I don't know what supported_devstudio_is_version is but we need to fix the version in Lina 77 and the release date at least.

Comment (nickboldt) [inline _config/products.yml]: https://devstudio.redhat.com/9.0/stable/updates/integration-stack/compositeContent.xml ... Alpha2 is still the latest available Integration Stack version, but I'll fix the other typos. Thanks!

File: _config/products.yml [hunk 2]
```diff
@@ -74,10 +74,36 @@ devstudio:
...
-      9.1.0.Beta1:
+      9.1.0.Beta2:
         update_site_url: https://devstudio.redhat.com/9.0/development/updates/
         #marketplace_url: https://marketplace.eclipse.org/node/1616973
         #marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2016-02-02
+        supported_devstudio_is_version: 9.0.0.Alpha2
+        documentation_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        release_notes_url: https://access.redhat.com/site/documentation/en-US/Red_Hat_JBoss_Developer_Studio/
+        blog_announcement_url: /blog/beta2-for-mars2.adoc
```
Comment (alexeykazakov) [inline _config/products.yml]: Should be .html. I will fix that.

**Comment by alexeykazakov** [comment]
Pushed to master and production after rebase and the fixed blog link.

**Pull request closed without merging**